### DISCUSSION TITLE
Google Public CA ECDSA Certs!

### DISCUSSION THREAD

**Discussion by IzumiSenaSora** [acmesh-official/acme.sh#4261] [Q&A]
Hi! 
I am using Google Public CA but its always get RSA certs! Even when i use ec-384 key ... is there any way to get ECDSA certs from Google Public CA?

**Comment by neilpang** [comment]
```
acme.sh --list
```

**Reply by IzumiSenaSora** [reply]
Sorry! I am bad at English! 
`--list` shows list of certs! 
I want to get ECDSA certs from different chain like 
Letsencrypt (ISRG Root X2) which provides ECDSA certs but Google Public CA always give me RSA Certs!
Is there any way to change chain of Google Public CA so i can request ECDSA Certs?
